contributors/join-us/staff-recruiting/README.md

```diff
@@ -70,3 +109,73 @@ Company pre-meet scores are as follows:
+
+### Mid-Interview Sync
+Ahead of MLT and AA interviews, the Recruiter may schedule a live mid-interview sync to align on intent to move forward and any ICP criteria that need additional assessment.  If there are multiple candidates in the pipe, the Recruiter may ask the interview panel to rank the candidate pool. This exercise helps clear any potential blindspots or confusion around role expectations. It also allows the Recruiting team to prioritize the highest ranking candidates to complete the interview process, as appropriate. 
+
+**Logistics**
+* Mid-interview sync will be pre-scheduled to occur after the last mid-interviewer completes. 
+* If there are multiple candidates for the same role, the sync may be scheduled after all candidates have gone through the last mid-interviewer. 
+* If you are an interviewer and cannot make the scheduled sync, please make sure to submit complete feedback in Lever ahead of the scheduled sync. 
+
+### Interview Project
+Candidates may be asked to complete a project as part of the interview process. Hiring Managers will indicate this need during the in-take. Hiring Managers can create their own project. Reach out to Recruiting for sample projects. 
+
+All take home projects should list the expected amount of time to complete. This provides candidates with expectation clarity. The Hiring Manager will indicate which stage of the recruiting process a take home will be given. The process for this stage will be standard for all roles: 
+
+1. Hiring Manager or Recruiter will inform the candidate of the Take Home project details and ask the candidate to complete either: 
+ * [CLA](https://mattermost.org/mattermost-contributor-agreement/)if the Take Home project is a Community Ticket 
+ * NDA if the above doesn’t apply 
+2. Hiring Manager or Recruiter will send the candidate the Take Home instructions through Lever. 
+3. Upon completion, the Hiring Manager or Recruiter will complete the [Interview Project Gift Card Form](https://docs.google.com/forms/d/e/1FAIpQLSfMppjsYmEyrMmY3tXnKdAs_cvA21o0vHRcyuVN6uFOoU3_Eg/viewform) which will notify the People Team to send a Thank You gift to the candidate for completing the project. 
+
+## Offer Stage
+
+### Reference Checks 
+WIP 
+
+### Offer Approval Process 
+Once reference checks are complete and an offer is ready to be made. The Recruiter will enter in the offer information in Lever for approval. 
+
+The order of approval will follow: 
+1. HR Director 
+2. MLT team member 
+3. VP, Finance & Operations 
+4. CEO 
+
+After offer approval, the Recruiter, Team MLT or AA Interview will extend a verbal offer to the candidate. Upon verbal acceptance, the Director of HR will send the offer letter. 
+
+### Extending an Offer 
+
+Recruiting will typically extend an offer, unless otherwise determined by the MLT team member or Hiring Manager. Exceptions may be made if the Team MLT memberor AA Interviewer prefers to extend the offer. 
```

Comment:
```suggestion
Recruiting will typically extend an offer, unless otherwise determined by the MLT member or Hiring Manager. Exceptions may be made if the Team MLT member or AA Interviewer prefers to extend the offer.
```